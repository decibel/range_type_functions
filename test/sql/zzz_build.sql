\set ECHO none

\i test/pgxntool/psql.sql

BEGIN;
-- NOTE! This is NOT the .sql file!
\i sql/range_type_functions.sql

\echo # TRANSACTION INTENTIONALLY LEFT OPEN!

-- vi: expandtab sw=2 ts=2
