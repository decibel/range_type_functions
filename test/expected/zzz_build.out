\set ECHO none
            _range_from_array__create            
-------------------------------------------------
 range_from_array(integer[])
 range_from_array(numeric[])
 range_from_array(timestamp without time zone[])
 range_from_array(timestamp with time zone[])
 range_from_array(date[])
 range_from_array(bigint[])
(6 rows)

# TRANSACTION INTENTIONALLY LEFT OPEN!
